lua/cmake/init.lua

```diff
@@ -164,6 +177,24 @@ function cmake.select_build_type()
   end)
 end
 
+function cmake.select_dap_config()
+  local project_config = ProjectConfig.new()
+  local dap_configs = {}
+  local n = 1
+  for k, _ in pairs(config.dap_configurations) do
+    dap_configs[n] = k
+    n = n + 1
+  end
```

Comment:
There is a nice helper for it: `vim.tbl_keys`.

Reply:
I refactored that bit to make use of it